src/server/verification/verificationAPI.js

```diff
@@ -390,6 +391,94 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
     })
   )
 
+  /**
+   * @api {post} /verify/hanuka-bonus Check hanuka bonus availability
+   * @apiName Hanuka Bonus
+   * @apiGroup Verification
+   *
+   * @apiSuccess {Number} ok
+   * @ignore
+   */
+  app.get(
+    '/verify/hanuka-bonus',
+    passport.authenticate('jwt', { session: false }),
+    wrapAsync(async (req, res, next) => {
+      const log = req.log.child({ from: 'verificationAPI - verify/hanuka-bonus' })
+      const { user } = req
+      const now = moment().utcOffset('+0200')
+      const startHanuka = moment(`${conf.hanukaStartDay}/12`, 'DD/MM')
+      const endHanuka = moment(`${conf.hanukaEndDay}/12`, 'DD/MM').endOf('day')
+
+      log.info('Current date', now)
```

Comment:
put more thought into what you log as info/debug, merge logs

Reply:
Done